source/docs/software/advanced-controls/introduction/tuning-pid-controller.rst

```diff
@@ -1,32 +1,229 @@
 Tuning a PID Controller
 =======================
 
-These steps apply to position PID controllers. Velocity PID controllers typically don't need :math:`K_d`.
+As seen in :ref:`the introduction to PID controller <docs/software/advanced-controls/introduction/introduction-to-pid:Introduction to PID>`, a PID controller has three *tuned* constants. The numeric values for these constants must be picked carefully for the specific mechanism under control. These constants will generally have different values on different robots. 
 
-1. Set :math:`K_p`, :math:`K_i`, and :math:`K_d` to zero.
+There are multiple methods for determining the values for these constants on your particular mechanism.
+
+The :ref:`SysId toolsuite <docs/software/pathplanning/system-identification/index:System Identification>` can be used to model your system and give accurate Proportional and Derivative values. This is preferred for supported mechanism types.
+
+Manual PID Tuning
+-----------------
+
+In this section, we'll go through some techniques to manually find reasonable values for the gains in a PID controller.
+
+This is useful if you are not using the :ref:`SysId toolsuite <docs/software/pathplanning/system-identification/index:System Identification>`. Additionally, even if you are using it, it is useful to see and understand the behavior of changing the values of the constants in different situations.
+
+Mechanism Walkthrough - Flywheel 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For this walkthrough, use this interactive simulation to explore tuning concepts:
+
+.. raw:: html
+
+    <div class="viz-div">
+      <div class="col" id="flywheel_pid_plot"></div>
+      <div class="flex-grid">
+         <div class="col" id="flywheel_pid_viz"></div> 
+         <div id="flywheel_pid_ctrls"></div>
+      </div>
+      <script>
+         flywheel_pid = new FlywheelPIDF("flywheel_pid");  
+         flywheel_pid.runSim(); 
+      </script> 
+    </div>
+
+Mechanism Description
+~~~~~~~~~~~~~~~~~~~~~
+The "Flywheel" is nothing more than a circular mass, attached to a gearbox, and driven by a motor. They're commonly used to propel game pieces through the air, into a target. 
+
+In general: the more voltage that is applied to the motor, the faster the flywheel will spin. Once voltage is removed, friction slowly decreases the spinning until the flywheel stops.
+
+To consistently launch a gamepiece, a good first step is to make sure it is spinning at a particular speed before putting a gamepiece into it. 
+
+This design drives the controls goal we will use in this example: Put the correct amount of voltage into the motor to get the flywheel to a certain speed, and then keep it there.
+
+As a test, a gamepiece is injected into the flywheel about halfway through the simulation.
```

Comment:
this needs a footnote explaining the actual loading simulation for curious students